frontend/components/Editor/Editor.tsx

```diff
@@ -73,8 +84,41 @@ const Editor = ({
   const classNames = classnames(baseClass, className, {
     "form-field": isFormField,
     [`${baseClass}__error`]: !!error,
+    [`${baseClass}__wrapper--copy-enabled`]: enableCopy,
   });
 
+  const [showCopiedMessage, setShowCopiedMessage] = useState(false);
+
+  const renderCopyButton = () => {
+    const copyValue = (e: React.MouseEvent) => {
+      e.preventDefault();
+      stringToClipboard(value).then(() => {
```

Comment:
Interesting that the underlying method here returns a Promise.

Would be slightly better to define this outside of `renderCopyButton`

Reply:
Oh yeah, fixing it here and improving the same on `<InputField/>`